boards/sparc/bm3823/xx3823/configs/nsh/defconfig

```diff
@@ -62,6 +54,3 @@ CONFIG_TASK_SPAWN_DEFAULT_STACKSIZE=768
 CONFIG_UART1_BAUD=9600
 CONFIG_UART1_SERIAL_CONSOLE=y
 CONFIG_UART2_BAUD=9600
-CONFIG_USERMAIN_STACKSIZE=4096
-CONFIG_USER_ENTRYPOINT="nsh_main"
```

Comment:
CONFIG_INIT_ENTRYPOINT="nsh_main" and move to the right location

Reply:
Done